msgcntr/messageforums-app/src/java/org/sakaiproject/tool/messageforums/ForumsEmailService.java

```diff
@@ -345,4 +348,30 @@ private boolean isValidEmail(String email) {
 		
 	}
 
+	private String getAnonAwareAuthor(Message message)
+	{
+		if (isUseAnonymousId(message.getTopic()))
```

Comment:
Should the `message` object be null checked here? I'm not sure if it's possible for it to be null.